casper/src/test/scala/coop/rchain/casper/ManyValidatorsTest.scala

```diff
@@ -31,7 +31,7 @@ class ManyValidatorsTest
     implicit val log       = new Log.NOPLog[Task]()
     val bonds = Seq
       .fill(300)(
-        ByteString.copyFromUtf8(Random.nextString(20)).substring(0, 32)
+        ByteString.copyFromUtf8(Random.nextString(30)).substring(0, 65)
```

Comment:
why 30?

Reply:
I believe that is because on average a random 30 character long utf8 string consists out of 80-90 bytes. That is a very strange way to generate 65 bytes though. I don't know why we are not using just `Random.nextBytes(Array.ofDim[Byte](65))`.